frame/contracts/proc-macro/src/lib.rs

```diff
@@ -162,6 +162,8 @@ struct HostFn {
 	returns: HostFnReturn,
 	is_stable: bool,
 	alias_to: Option<String>,
+	/// Formulating the predicate inverted makes the expression using it simpler.
+	is_undeprecated: bool,
```

Comment:
I don't think that `undeprecated` exists as a word. How about `not_deprecated`?

Reply:
Yes. Good idea :)